PWT/Roxie/Tournament.yaml

```diff
@@ -0,0 +1,58 @@
+species: Toxicroak
+setname: PWT-Roxie Tournament
+tags: [in-game, PWT-Roxie Tournament, trainer+roxie]
+item: [Focus Sash]
+ability: [Anticipation, Dry Skin]
+moves:
+- [Protect]
+- [Drain Punch]
+- [Poison Jab]
+- [Swords Dance]
+gender: [m, f]
+happiness: 255
+biddable: false
+ball: [Poke]
+nature: Serious
+ivs: {hp: 0, atk: 0, def: 0, spA: 0, spD: 0, spe: 0}
+evs: {hp: 0, atk: 0, def: 0, spA: 0, spD: 0, spe: 0}
+level: 100
+rarity: 1.0
+---
+species: Crobat
+setname: PWT-Roxie Tournament
+tags: [in-game, PWT-Roxie Tournament, trainer+roxie]
+item: [Psychic Gem]
```

Comment:
gem

Reply:
Another set prolly goin in the trash then. Darn